spec/models/availability_zone_spec.rb

```diff
@@ -9,4 +9,10 @@
     described_class.available.length.should == 2
     described_class.available.each { |az| az.class.should_not == ManageIQ::Providers::Openstack::CloudManager::AvailabilityZoneNull }
   end
+
+  it ".event_where_clause" do
```

Comment:
@gtanzillo just to be clear, this is not exercising the specific issue sadly (which instead requires a MiqReport, rbac and other things to be involved).